pkg/square/square.go

```diff
@@ -93,8 +93,8 @@ func (s Square) Size() int {
 	return Size(len(s))
 }
 
-func Size(len int) uint64 {
-	return shares.RoundUpPowerOfTwo(uint64(math.Ceil(math.Sqrt(float64(len)))))
+func Size(len int) int {
+	return int(math.Ceil(math.Sqrt(float64(len))))
```

Comment:
The merge commit no longer rounds up, do we still want it @evan-forbes ? Related: https://github.com/celestiaorg/celestia-app/pull/1761#discussion_r1192889489

Reply:
yeah thanks good catch, how this wasn't a conflict idek. yeah we need it in order to get tests to pass here and on the branches based on this one